config/config.devnet.yaml

```diff
@@ -35,6 +35,12 @@ features:
     enabled: false
     hitsThreshold: 100
     ttl: 12
+  transactionPool:
+    enabled: true
+  transactionPoolWarmer:
+    enabled: true
+    cronExpression: '*/5 * * * * *'
+    ttlInSeconds: 6
```

Comment:
default ttl should be maybe 60 seconds, to make sure the value doesn't expire if the gateway experiences a slowdown

Reply:
done